versions/3.0.md

```diff
@@ -1152,6 +1152,51 @@ name: pet
 description: Pets operations
 ```
 
+#### <a name="examplesObject"></a>Examples Object
+
+Anywhere an `example` may be given, allow a $ref object.  This does mean that `example` can be either a string primitive or an object, like `additionalProperties`:
```

Comment:
@webron Maybe I'm misunderstanding this. [Example Object (for responses)](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#exampleObject):

> The name of the property MUST be one of the Operation produces values (either
> implicit or inherited). The value SHOULD be an example of what such a response
> would look like.

The example for this (with a JSON object) seem to indicate that this is an instance matching the schema.

[In the schema object](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#schemaObject) we have:

Field Name | Type | Description
--------------|--------|-----------
example | Any | A free-form property to include a an example of an instance for this schema.

Also here I would understand (and the examples agree) that this usually include the JSON value, not a string representation thereof. (While it might not be a "constraint", given the type "Any", the text + examples strongly suggest this.)